sotodlib/mapmaking/demod_mapmaker.py

```diff
@@ -523,6 +523,7 @@ def make_demod_map(context, obslist, noise_model, info,
         List of outputs from preprocess database. To be used in cleanup_mandb.
     """
     from .. import site_pipeline
```

Comment:
This import here is only for the logger. @mhasself does not want us to import the site pipeline stuff as if it were a library. On the other hand, this logger is needed here because passing a logger from the script side to this function won't work with multiprocessing, as it won't print messages. Hence on this function, we have to create a new logger object to be able to print messages (this is how @msilvafe described it to me). Is there a workaround to this? Any suggestions? Maybe the logger stuff should also be moved to the library side.

Reply:
We've copied the logger stuff into preprocess/preprocess_util.py so that the preprocessing module also has its own logger available.  The mapmaking module could use that one or it could copy the logger functions into its own util file of some kind maybe.

Reply:
ok, perfect. Can you change the logger to be created from preprocess_util then, please?